### PR TITLE
Makes upload_set_id available in the WorkShowPresenter. 

### PR DESCRIPTION
**Pull request by hectorcorrea**
Fixes https://github.com/projecthydra-labs/sufia-core/issues/389

### CODE REVIEW

**Comment by awead** [comment]
:+1:

**Comment by mjgiarlo** [comment]
Travis is :unamused:

**Comment by jcoyne** [comment]
Rebase this and Travis should be happy

File: app/presenters/sufia/work_show_presenter.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ module Sufia
   class WorkShowPresenter < ::CurationConcerns::WorkShowPresenter
     # delegate fields from Sufia::Works::Metadata to solr_document
     delegate :based_near, :depositor, :identifier, :resource_type, :tag, to: :solr_document
+    attr_reader :upload_set_id
```
Comment (grosscol) [inline app/presenters/sufia/work_show_presenter.rb]: Where does this attribute get assigned a value?  Should this get indexed into the Work's solr_doc and delegated?  We'll have to address the workflow where a Work is the recipient of multiple UploadSets, but that is for future us to sort out.

**Pull request closed without merging**